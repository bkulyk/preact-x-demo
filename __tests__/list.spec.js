import
React from 'react';
import List from '@bitchin/list';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(shallow(<List>some-children</List>)).toMatchSnapshot();
});

it('should be able to render a different component', () => {
  expect(shallow((
    <List
      component="div"
      className="custom"
      data-something="something"
    >
      some-children
    </List>
  ))).toMatchSnapshot();
});
