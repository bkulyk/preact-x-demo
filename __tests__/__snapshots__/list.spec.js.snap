// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be able to render a different component 1`] = `
<div
  className="mdc-list custom"
  data-something="something"
>
  some-children
</div>
`;

exports[`should render correctly 1`] = `
<ul
  className="mdc-list"
>
  some-children
</ul>
`;
